test/Chirp.Infrastructure.Tests/CheepRepositoryCreateTest.cs

```diff
@@ -105,4 +105,16 @@ public void UnitTestCreateMethodAddedCheepShouldNotBeInDatabase2()
         //Should throw an exception to pass
         act.Should().ThrowAsync<ValidationException>().WithMessage("Exception of type 'System.ComponentModel.DataAnnotations.ValidationException' was thrown.");
     }
+
+    //Test that deleting all of an authors cheeps works
+    [Fact]
+    public void UnitTestDeleteCheepsFromAuthor()
+    {
+        //Act
+        repository.DeleteCheepsFromAuthor(1);
+
+        //Assert
+        //Should pass since the cheeps are deleted
+        _context.Cheeps.Should().BeEmpty();
```

Comment:
I have a question:

Does this check if there are no cheeps in the database for that author, or that there is no cheeps at all in the database.

If it is the later, we should probarbly change the test to match the first described condition.
Otherwise, could you please explain how it asserts only cheeps for the author with id one.

Reply:
There Should be more than one Cheep in the set to ensure it does in fact work as intended